tutor/src/models/tour/region.js

```diff
@@ -15,7 +14,7 @@ export default class TourRegion extends BaseModel {
   @field({ type: 'array' }) otherTours;
 
   @computed get tour_ids() {
-    return concat( [this.id], this.otherTours.peek() );
+    return this.otherTours.reverse().concat( [this.id] ).reverse();
   }
```

Comment:
hm, I actually had to employ the console to figure this out :P 
`[ 1, 2, 3 ].reverse().concat(['a']).reverse()` yields: `["a", 1, 2, 3]`

Seems like the: `['a'].concat([ 1, 2, 3 ])` is easier to grok, but maybe that's just me.  

The peek() is needed since `otherTours` isn't areal array, but it's a non-copy operation (`slice` copies) so should be efficient https://mobx.js.org/refguide/array.html